src/components/Footer.js

```diff
@@ -0,0 +1,115 @@
+import React from "react";
+import {
+  Address,
+  Footer as FooterUSWDS,
+  FooterNav,
+  Grid,
+  Logo,
+  SocialLinks,
+} from "@trussworks/react-uswds";
+import { useTranslation } from "react-i18next";
+import Link from "../components/Link";
+
+import logoImg from "../../public/static/favicon.ico";
+
+function Footer() {
+  const { t } = useTranslation();
+  const primaryLinks = ["#", "#", "#", "#"];
+
+  return (
+    <FooterUSWDS
+      size="medium"
+      // returnToTop={returnToTop}
+      primary={
+        <FooterNav
+          size="medium"
+          links={Array.from({ length: 4 }, (_, i) => {
```

Comment:
Can we not hardcode the array length here?

Reply:
Not hardcoded anymore, check newest commit.